draw.typ

```diff
@@ -231,8 +231,11 @@
   ),)
 }
 
-#let arc(position, start, stop, radius: 1, mode: "OPEN", name: none, anchor: none, fill: auto, stroke: auto) = {
+#let arc(position, start: auto, stop: auto, delta: auto, radius: 1, mode: "OPEN", name: none, anchor: none, fill: auto, stroke: auto) = {
+  assert( ((start != auto and (stop != auto or delta != auto)) or (stop != auto and delta != auto)) and (start == auto or stop == auto or delta == auto), message: "Exactly two of three options start, auto and delta should be defined")
```

Comment:
I think a less confusing way to check would be to use filter to get all the `auto`s and make sure there is only one. Also change "auto" to "stop" in the message